### DISCUSSION TITLE
How can I get GraphQL Schema

### DISCUSSION THREAD

**Discussion by aguragorn** [nocodb/nocodb#1448] [Q&A] [answered]
I deployed via 1-click heroku deploy. I am trying the graphql using the UI and now I want to try to query from outside. How do I get the schema, I wanted to use Apollo Kotlin to help me query via graphQL

**Comment by aguragorn** [comment, marked as answer]
Nevermind, Apollo Kotlin has a good documentation on how to get the schema from the graphql server
https://www.apollographql.com/docs/kotlin/tutorial/02-add-the-graphql-schema#download-your-servers-schema

**Reply by o1lab** [reply]
Awesome, this will help others
